### PR TITLE
Add Opencast and Postiz to Hacktoberfest swag list

### PR DESCRIPTION
**Pull request by nabeelmohamed**
Thanks for contributing to the [Hacktoberfest Swag List](https://hacktoberfestswaglist.com/) :smiley: :tada:! Before submitting your pull request, please check off as many of the items below as you can:

1. [x] I have read the [Contributing.md](https://github.com/crweiner/hacktoberfest-swag-list/blob/master/docs/contributing.md) file and formatted this PR correctly
2. [x] I'm not adding a company from the blocklist
3. [x] I make sure to fix things promptly if an error or suggestion comes up

Thanks and Happy Hacktoberfest! :tada:
Tagging @crweiner to take a look. :eyes:

### CODE REVIEW

**Comment by nabeelmohamed** [comment]
typo error *Owncast*

**Comment by vimode** [comment]
Owncast doesn't seem to be providing exclusive swag for _hacktoberfest_. As far as I remember this list includes only the companies and projects which give swag exclusively for contributions during _hacktoberfest_. 

As per the issues at Owncast,
> Active contributors get an Owncast t-shirt! Ask about it if you feel like you've been contributing and haven't yet been given one.

@crweiner IIRC, isn't this the same reason why Kong is not included in the list anymore?

**Comment by nabeelmohamed** [comment]
@vimode  you can find the swag details here( https://owncast.online/hacktoberfest/#content-no-code-or-low-code-contributions)

**Comment by vimode** [comment]
@nabeelmohamed Thank you for confirming their hacktoberfest participation with an appropriate link!
This link should be included in it too for contributors to read about it. 
If you check the other orgs added to the list, you will get an idea where and how it should be added. 
And please do check all the links which are included in the PR to make sure they are not dead. The contributing link is.

File: docs/list.md [hunk 1]
```diff
@@ -341,6 +341,31 @@ MIT-licensed open-source
...
+#### **Owncast**
+
+- **Swag**: Digital badge, Owncast [t-shirt](https://gabekangas.typeform.com/to/Wfx9UHVZ?typeform-source=owncast.online), the contributors thank-you video on [Owncast TV](https://watch.owncast.online/)
+- **Requirements**:
+    - Register for Hacktoberfest 2024 by clicking [here](https://hacktoberfest.com/participation/)
+    - Make quality contributions to Owncast (e.g. fix a bug, write useful documentation, create content around Owncast)
+    - Get 4 pull/merge requests accepted during Hacktoberfest to level up your digital badge
+- **How to sign up**: No seperate signup known
+- **Issues**: Look for [Good First Issues](https://github.com/owncast/owncast/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22)
+- **Notes**: Read the [Owncast Contributor Guide](https://owncast.online/docs/contributing/) and for more info check [here](https://owncast.online/hacktoberfest/#content-no-code-or-low-code-contributions)
```
Comment (vimode) [inline docs/list.md:353]:
```suggestion
- **Swag**: T-shirt.
- **Requirements**:
	- Make quality code or non-code or low code contributions to Owncast (e.g. fix a bug, write useful documentation, create content around Owncast).
	- After contributing, [fill this form to request a T-shirt.](https://gabekangas.typeform.com/to/Wfx9UHVZ)
- **How to sign up**: No separate signup known.
- **Issues**: All eligible [open issues.](https://github.com/owncast/owncast/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22)
- **Notes**: Read the [Owncast Contributing Guidelines](https://owncast.online/hacktoberfest/#what-do-you-receive-by-contributing-to-owncast-during-hacktoberfest) for more information.
```

File: docs/list.md [hunk 2]
```diff
@@ -341,6 +341,31 @@ MIT-licensed open-source
...
+
+- **Swag**: T-shirt for every merged PR (max 1 per person).
+- **Requirements**:
+  - You must create an issue before making a pull request.
+  - You can also ask to be assigned to an issue. During Hacktoberfest, each issue can have multiple assignees.
+  - We have to approve the issue and add a "hacktoberfest" tag.
+  - We encourage everybody to contribute to all types of issues. We will only send swag for issues with features and bug fixes (no typos, sorry).
+- **How to sign up**: No separate signup known
+- **Issues**: All [open issues](https://github.com/gitroomhq/postiz-app/issues)
+- **Notes**: Read the [contributing guide](https://github.com/gitroomhq/postiz-app/blob/main/CONTRIBUTING.md) for more information.
```
Comment (vimode) [inline docs/list.md:367]:
```suggestion
- **Swag**: T-shirt.
- **Requirements**:
  - You must create an issue before making a pull request.
  - You can also ask to be assigned to an issue. During Hacktoberfest, each issue can have multiple assignees.
  - We have to approve the issue and add a "hacktoberfest" tag.
  - We encourage everybody to contribute to all types of issues. We will only send swag for issues with features and bug fixes (no typos, sorry).
- **How to sign up**: No separate signup known
- **Issues**: All [open issues](https://github.com/gitroomhq/postiz-app/issues)
- **Notes**: Read the [announcement post](https://github.com/gitroomhq/postiz-app#we-participate-in-hacktoberfest-2024-) for more information.
```

**Review by vimode — CHANGES_REQUESTED** [review]
Suggested some changes.

**Comment by crweiner** [comment]
Hi @nabeelmohamed  and @vimode! Thank you for adding Opencast and Postiz to the Hacktoberfest Swag List! I love seeing new organizations participate! I've made some tweaks and am happy to get this live.

Thanks again!

**Pull request merged**